### PR TITLE
ADD: file logger

### LINKED ISSUE #80: Log to file in production

### PR DESCRIPTION
**Pull request by Segelzwerg**
closes #80

### CODE REVIEW

File: family_foto/logger.py [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
 from logging.config import dictConfig
 
+LOG_FILE_NAME = 'server_werkzeug.log'
+LOG_PATH = 'instance/log'
+
+if not os.path.exists(LOG_PATH):
+    os.mkdir(LOG_PATH)
+
+file = open(f'{LOG_PATH}/{LOG_FILE_NAME}', 'w+')
+file.close()
```
Comment (iTitus) [inline family_foto/logger.py:11]: ???
what are you doing here?
if you want to create a empty file do `touch`.
if you want to check permissions wrap this into a try-except

Comment (Segelzwerg) [reply family_foto/logger.py:11]: Oh yeah, I was mixing up commands and forgot, I could do this

Comment (Segelzwerg) [reply family_foto/logger.py]: I think there is no `touch` in python, but I shortened it down.

File: family_foto/logger.py [hunk 2]
```diff
@@ -1,19 +1,40 @@
 import logging
+import os
 from logging.config import dictConfig
 
+LOG_FILE_NAME = 'server_werkzeug.log'
+LOG_PATH = os.path.join('instance', 'log')
+
+if not os.path.exists(LOG_PATH):
+    os.mkdir('instance')
+    os.mkdir(LOG_PATH)
```
Comment (iTitus) [inline family_foto/logger.py:10]:
```suggestion
    os.makedirs(LOG_PATH)
```

**Review by iTitus — CHANGES_REQUESTED** [review]

**Review by iTitus — APPROVED** [review]

**Pull request merged**